MAINTAINERS.md

```diff
@@ -0,0 +1,9 @@
+# Maintainers
+
+Current active maintainers of this SDK:
+
+- [Lance Ball](https://github.com/lance)
+- [Daniel Bevenius](https://github.com/danbev)
+- [Lucas Holmquist](https://github.com/lholmquist)
+- [Fabio Jose](https://github.com/fabiojose)
+- [Helio Frota](https://github.com/helio-frota)
```

Comment:
I'm pretty sure myself and lance would be the only two as active maintainers at this point

Reply:
I was just going by what i see here: https://github.com/orgs/cloudevents/teams/sdk-javascript-maintainers
If you'd like you could start a vote to reduce it down to just you two per: https://github.com/cloudevents/spec/blob/main/docs/SDK-GOVERNANCE.md

Reply:
sure